.coveragerc

```diff
@@ -4,10 +4,7 @@ source = gocept.bbissues
 
 [report]
 precision = 2
-omit = */interfaces.py
-       */interfaces/*
-       */tests.py
-       */tests/*
+fail_under = 44
```

Comment:
coveralls actually shows 45.5%.

Reply:
Locally running `tox -ecoverage` I just get 44.83 %. I do not know what's different on coveralls.io. (But I had the same before vice versa for zopefoundation packages.)